index.html

```diff
@@ -97,32 +117,226 @@
 					        	<a class="nav-link" href="#">נקודות שירות
 					        	</a>
 					        </li>
-					        <li class="nav-item active">
-					        	<a class="nav-link" href="#">אודות
-					        	</a>
-					        </li>
 			    		</ul>
-	    			</div>
-	    			<div class="paz-logo-nav-2">
-					        	<img src="img/paz-logo-nav-2.png">
+			    	</div>
+		    		<div class="paz-logo-nav-2">
+				        <img src="img/pazomat-logo.svg"
+    					 class="pazomat-logo">
 					</div>
-	    		</div>
-	    	</div>
-    </header>
-   	
-    	
-
-
-
-
-
-
-
-
-    	<!-- TOP SECTION -->
-    	<section>
+    			</div>
+			</div>
+		</div>
+	</header>
+	
+	<!-- TOP SECTION -->
+	<section class="container top-section">
+		<div class="gallery">
+			<div class="Locating-stations">
+				<div class="swiper-container">
+				    <!-- Additional required wrapper -->
+				    <div class="swiper-wrapper">
+				        <!-- Slides -->
+				        <div class="swiper-slide"><img src="img/main-gallery-slide-1.png"></div>
+				        <div class="swiper-slide"><img src="img/main-gallery-slide-1.png"></div>
+				        <div class="swiper-slide"><img src="img/main-gallery-slide-1.png"></div>
+				        ...
+				    </div>
+				    <!-- If we need pagination -->
+				    <div class="swiper-pagination"></div>
+				 
+				    <!-- If we need navigation buttons -->
+				    <div class="swiper-button-prev"></div>
+				    <div class="swiper-button-next"></div>
+				 
+				    <!-- If we need scrollbar -->
```

Comment:
You can remove all these swiper comments... You pasted them in here but you don't need them